formats/ome-tiff/specification.txt

```diff
@@ -1,26 +1,27 @@
 OME-TIFF specification
 ======================
 
-
 The following provides technical details on the OME-TIFF
 format. It assumes familiarity with both the
 `TIFF <http://partners.adobe.com/public/developer/tiff/index.html>`_ and
 :schema_plone:`OME-XML <>` formats, although there is some review of both.
 
-Storing OME-XML within a TIFF
------------------------------
+An OME-TIFF dataset consists of;
+
+- one or more files in standard TIFF format with the file extension
+  ``.ome.tif`` or ``.ome.tiff`` or `BigTIFF format <http://bigtiff.org/>`_
+  with the file extension ``.ome.tf2``, ``.ome.tf8`` or ``.ome.btf``
+- a string of OME-XML metadata embedded in the ImageDescription tag of each
+  file first IFD (Image File Directory).
 
-An OME-TIFF dataset consists of one or more files in standard TIFF or
-`BigTIFF format <http://bigtiff.org/>`_, with the file extension
-``.ome.tif`` or ``.ome.tiff``, and an identical (or in the case of
-multiple files, nearly identical) string of OME-XML metadata embedded
-in the ImageDescription tag of each file's first IFD (Image File
-Directory).  BigTIFF file extensions are also permitted, with the file
-extension ``.ome.tf2``, ``.ome.tf8`` or ``.ome.btf``, but note these
+BigTIFF file extensions are also permitted, with the file extension ``.ome.tf2``, ``.ome.tf8`` or ``.ome.btf``, but note these
```

Comment:
This is partially duplicating the detail above; the file extensions could be dropped.